src/state.js

```diff
@@ -28,8 +28,14 @@ export default {
     _guesses: [],
     wins: 0,
     score: 0,
+    songs: [],
 
     async init() {
+        const result = await fetch('playpass-content.json');
```

Comment:
instead of doing raw fetching in the template, do we want to revisit making this sdk functionality?

https://github.com/playpassgames/playpass/pull/20